### ISSUE TITLE
artifacts | exclude doesn't work

### ISSUE THREAD

**Issue by d34dman** [factorial-io/phabalicious#73]
When using artifact deployment (git), we found that the code gets stuck on exclude statement.

Docuementation link : https://factorial-io.github.io/phabalicious/deploying-artifacts.html#exclude

```
hosts:
  artifact-foo:
    branch: develop
    artifact:
      actions:
        - action: exclude
          arguments:
            - "sites/all/modules/custom/foo_forms"
```

here is the debug output in console during that phase,

```
[debug] Running task appCreatePrepare on configuration artifact-foo
[debug] Running task appCreate on configuration artifact-foo
[debug] Call task appCreate on method git
[debug] Call task appCreate on method artifacts--git
[debug] New working dir: /tmp/artifact-foo-target-1583420637
[notice] cd /tmp/artifact-foo-target-1583420637 && #! git checkout sites/all/modules/custom/foo_forms
```

The workaround we did was to use `script` and run `git checkout sites/all/modules/custom/foo_forms`. So it seems the `#! ` (shebang with a space) is causing the problem.

**Comment by stmh** [comment]
please test the latest dev-version

**Comment by stmh** [comment]
FIxed in 3.4.0
